onnx/backend/test/data/real/test_zfnet512/data.json

```diff
@@ -1 +1 @@
-{"atol": 1e-07, "model_name": "zfnet512", "rtol": 0.001, "url": "https://s3.amazonaws.com/download.onnx/models/opset_9/zfnet512.tar.gz"}
```

Comment:
Based on the specified opset version here: https://github.com/onnx/onnx/blob/7f12395a87cc85d9c6138a8b99530571014d3307/onnx/backend/test/case/model/__init__.py#L40, updating opset_10 here should be correct. I was surprise that these haven't been updated for a while. Since now we do test all models from ONNX model Zoo weekly, perhaps we can consider to remove/refactor these old models.